fuzzers/nyx_libxml2_parallel/setup_libxml2.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+cargo build --release
+export CC=`pwd`/target/release/libafl_cc
+export CXX=`pwd`/target/release/libafl_cxx
+curl -C https://gitlab.gnome.org/GNOME/libxml2/-/archive/v2.9.14/libxml2-v2.9.14.tar.gz --output libxml2-v2.9.14.tar.gz
+tar -xvf ./libxml2-v2.9.14.tar.gz  --transform s/libxml2-v2.9.14/libxml2/
+cd ./libxml2/ || exit
+./autogen.sh --enable-shared=no
+make
+cd - || exit
+python3 "../../libafl_nyx/packer/packer/nyx_packer.py" \
+    ./libxml2/xmllint \
+    /tmp/nyx_libxml2 \
+    afl \
+    instrumentation \
+    -args "/tmp/input" \
+    -file "/tmp/input" \
+    --fast_reload_mode \
+    --purge
+
+python3 ../../libafl_nyx/packer/packer/nyx_config_gen.py /tmp/nyx_libxml2/ Kernel
+sudo modprobe -r kvm-intel # or kvm-amd for AMD 
```

Comment:
can we use lsmod to check which one is available?
not everyone is using intel (I'm not 😅 )

Reply:
ah true, we need intel pt

Reply:
I have added `cfg!(target_arch = "x64")` in `build.rs`

Reply:
I mean amd x64, but it's fine if it just does not work.